woocommerce-gateway-ebanx/services/class-wc-ebanx-checker.php

```diff
@@ -56,7 +66,7 @@ public static function check_merchant_api_keys($context)
 
 			$api_url = 'https://api.ebanx.com';
 
-			$message = sprintf('Could not connect to EBANX servers. Please check if your server can reach our API (<a href="%1$s">%1$s</a>) and your integrations keys are correct.', $api_url);
+			$message = sprintf('EBANX - Could not connect to our servers. Please check if your server can reach our API (<a href="%1$s">%1$s</a>) and your integrations keys are correct.', $api_url);
```

Comment:
We need a better text for this...
Like, check connection to our servers first and then have his keys checked.